### PR TITLE
Provided more information for adding an HTTP Proxy

### PR DESCRIPTION
**Pull request by ianf77**
A table of the hostnames available and portnumbers, plus a warning not to perform SSL inspections
on these communications was added.

Bug 1921928 - Satellite documentation needs more information on proxy part if it use to sync repositories

https://bugzilla.redhat.com/show_bug.cgi?id=1921928

Cherry-pick into:

* [X] Foreman 2.5 (Satellite 6.10)
* [X] Foreman 2.4
* [X] Foreman 2.3 (Satellite 6.9)
* [x] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Review by melcorr — CHANGES_REQUESTED** [review]

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc [hunk 1]
```diff
@@ -500,6 +500,20 @@ To use the CLI instead of the web UI, see the xref:cli-adding-a-new-http-proxy_{
...
++
+[cols="2,1,1",options="header"]
+|====
+| Host name | Port | Protocol
+| subscription.rhsm.redhat.com | 443 | HTTPS
+| subscription.rhn.redhat.com | 443 | HTTPS
+| cdn.redhat.com |  443 | HTTPS
+| *.akamaiedge.net | 443 | HTTPS
+| *.akamaitechnologies.com |  443 | HTTPS
+| api.access.redhat.com (if using Red{nbsp}Hat Insights) |  443 | HTTPS
```
[the remark this answers is not in the document]
Comment (melcorr) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:513]: lol typing too fast, sorry @ianf77 
Rule of thumb for the future: Anything that mentions Red Hat Insights should be wrapped with an ifdef that is only for Satellite.

**Comment by melcorr** [comment]
If you need any guidance, let me know.

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc [hunk 2]
```diff
@@ -493,13 +493,31 @@ Use this option if you have one of the following errors:
...
+[cols="2,1,1",options="header"]
+|====
+| Host name | Port | Protocol
+| subscription.rhsm.redhat.com | 443 | HTTPS
+| subscription.rhn.redhat.com | 443 | HTTPS
+| cdn.redhat.com |  443 | HTTPS
+| *.akamaiedge.net | 443 | HTTPS
+| *.akamaitechnologies.com |  443 | HTTPS
+ifdef::satellite[]
+| api.access.redhat.com (if using Red{nbsp}Hat Insights) |  443 | HTTPS
```
Comment (melcorr) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:516]: We use example.com rather than these fake names

**Review by melcorr — CHANGES_REQUESTED** [review]
Thanks for applying the changes. 
Please remove the akamaitechnologies.com and akamai etc

**Review by melcorr — APPROVED** [review]
LGTM

**Comment by melcorr** [comment]
To get this into versions pre-2.3, please raise a PR against the 2.3 branch because there were so many changes over the last few releases to content.

**Comment by ianf77** [comment]
Hi Melanie
I raised a new PR as requested, to be cherry picked to 2.3 and 2.1
https://github.com/theforeman/foreman-documentation/pull/567

**Pull request merged**